media/system/js/fields/submenu.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * @copyright  Copyright (C) 2005 - 2017 Open Source Matters, Inc. All rights reserved.
+ * @license    GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+(function() {
+	document.addEventListener('DOMContentLoaded', function() {
+
+		if (window.toggleSidebar) {
+			toggleSidebar(true);
```

Comment:
I found 0 occurrences of this in project, so we can probably remove it...

Reply:
This is the 2nd sidebar menu. We probably wont use it in core, but for BC purposes, I'd like to keep it.

Reply:
Please then put it in legacy folder